src/lm_polygraph/utils/manager.py

```diff
@@ -520,9 +524,11 @@ def calculate(self, batch_stats: dict, calculators: list, inp_texts: list) -> di
                     batch_stats[stat] = stat_value
             except Exception as e:
                 if self.ignore_exceptions:
-                    log_msg = f"Caught exception while calculating stats: {e} in Stat Calculator {stat_calculator}"
+                    lineno = e.__traceback__.tb_lineno
+                    log_msg = f"Caught exception while calculating stats: {e} in Stat Calculator {stat_calculator}, line {lineno}. Expect dependent estimator to fail.\n"
+                    sys.stderr.write("\n\n")
                     sys.stderr.write(log_msg)
-                    print(log_msg)
+                    sys.stderr.write(traceback.format_exc())
```

Comment:
If error occurs and we are ignoring exceptions, dump full error traceback and message about exception into stderr.